services/ui-src/src/components/menus/Menu.tsx

```diff
@@ -32,7 +32,7 @@ export const Menu = ({ handleLogout }: Props) => {
       </Box>
       <MenuList sx={sx.menuList} data-testid="menu-list">
         <MenuItem sx={sx.menuItem} data-testid="menu-option-manage-account">
-          <RouterLink to="/acct" alt="link to account page" tabindex={0}>
```

Comment:
is there anything else already at /acct that should be removed?

Reply:
nah, just a 404 page

Reply:
/acct was just a placeholder that didn't route to anything (read: routed to the 404 because nothing lives at /acct)